app/controllers/cuisine_controller.rb

```diff
@@ -0,0 +1,69 @@
+class CuisineController < ApplicationController
+
+  before_action :find_related_restaurants, :only => [:delete, :destroy]
+
+  def index
+    @cuisines = Cuisine.sorted
+  end
+
+  def show
+    @cuisine = Cuisine.find(params.require(:id))
+  end
+
+  def new
```

Comment:
is this action in use?

Reply:
Are you referring to the 'new' action? If so, then yes, it's in use (how else can I invent a whole new genre? :) )

Reply:
well people use the create action right below here ( i was wondering if anyone directly calls new here)
vvv